internal/login/command_test.go

```diff
@@ -904,21 +904,22 @@ func newLoginCmd(auth *ccloudv1mock.Auth, userInterface *ccloudv1mock.UserInterf
 	cfg := config.New()
 	var ccloudClientFactory *climock.CCloudClientFactory
 	var mdsClient *mdsv1.APIClient
```

Comment:
Should be defined inside the `if` since it's only used inside that scope

Reply:
It's used outside of the `if` in line 923

Reply:
`mdsClientManager` is needed there, but `mdsClient` isn't

Reply:
oh haha my bad, yeah.